modules/assim.batch/R/pda.mcmc.bs.R

```diff
@@ -188,9 +191,25 @@ pda.mcmc.bs <- function(settings, params.id = NULL, param.names = NULL, prior.id
         ## Read model outputs
         model.out <- pda.get.model.output(settings, run.id, bety, inputs)
         
-        ## Calculate likelihood (and store in database)
-        LL.new <- pda.calc.llik(settings, con, model.out, run.id, inputs, llik.fn)
+        # handle bias parameters if multiplicative Gaussian is listed in the likelihoods
+        if(any(unlist(any.mgauss) == "multipGauss")) {
+          isbias <- which(unlist(any.mgauss) == "multipGauss")
+          # testing now
+          nbias <- 1
+          bias.list <- return.bias(isbias, list(model.out), inputs, prior.list, nbias)
+          bias.terms <- bias.list$bias.params
+        } else {
+          bias.terms <- matrix(1, nrow = 1, ncol = 1) # just 1 for Gaussian
+        }
         
+        ## calculate error statistics      
```

Comment:
here I just work with optimum bias, should I have another bias specific propose/reject loop here (like in emulator where I can calculate SS for >1 bias per parameter set)?
```
chain
    iteration
        propose.parameters
        run.model
        get.model.output
            propose.bias (for a predefined number of times?)
            pda.calc.error
            pda.calc.llik.par
            pda.calc.llik
            accept/reject bias (store 1 bias)
        accept/reject model parameters

```